sage-ios/SAGE/SAGE/SchoolDetailViewController.swift

```diff
@@ -62,11 +63,14 @@ class SchoolDetailViewController: UITableViewController {
     func configureWithSchool(school: School) {
         self.title = school.name!
         AdminOperations.loadSchool(school.id, completion: { (updatedSchool) -> Void in
+            self.hideNoContentView()
             self.configureWithCompleteSchool(updatedSchool)
             self.activityIndicator.stopAnimating()
             self.schoolDetailHeaderView.mapView.hidden = false
 
-            }) { (message) -> Void in }
+            }) { (message) -> Void in
+                self.showNoContentView()
```

Comment:
hmm idk why the stop animating isnt here...oh actually can you add the self.refreshcontrol here too? it should be here too. it is okay to add the refresh in this PR